src/main/java/lt/vu/web/api/controller/packageSize/ListPackageSizeController.java

```diff
@@ -16,12 +19,15 @@
 
 @Path("/package-size")
 @RequestScoped
+@Api(value= "List package size ") // swagger minimum (controller name)
 public class ListPackageSizeController extends AbstractApiController<List<PackageSizeDTO>> {
 
     @Inject
     private PackageSizeRepository packageSizeRepository;
 
     @GET
+    @ApiOperation(value = "Retrieves valid package sizes",         //swagger optional (endpoint summary)
+                  notes = "Return some json to the client")  //swagger optional (description)
```

Comment:
No need for these redundant notes. I also tried adding `response` so it displays the model structure (which is the main point of having swagger) and it crashes.